content/doc/book/managing/plugins.adoc

```diff
@@ -4,13 +4,261 @@ layout: section
+meta-data from an Update Center, whether the primary Update Center operated by
+the Jenkins project
+footnoteref:[uc, https://updates.jenkins.io],
+or a custom Update Center.
+
+The plugins are packaged as self-contained `.hpi` files, which have all the
+necessary code, images, and other resources which the plugin needs to operate
+successfully.
+
+=== From the web UI
+
+The simplest and most common way for installing plugins is through the
+*Configure System* > *Manage Plugins* view, available to administrators of a
+Jenkins environment.
+
+Under the *Available* tab, plugins available for download from the configured
+Update Center can be searched and considered:
+
+image::managing/plugin-manager-available.png["Available tab in the Plugin Manager", role=center]
+
+Most plugins can be installed and used immediately by checking the box adjacent
+to the plugin and clicking *Install without restart*.
+
+
+[CAUTION]
+====
+If the list of available plugins is empty, the master might be incorrectly
+configured or has not yet downloaded plugin meta-data from the Update Center.
+Clicking the *Check now* button will force Jenkins to attempt to contact its
+configured Update Center.
+====
+
+[[install-with-cli]]
+=== Using the Jenkins CLI
+
+Administrators may also use the <<cli#,Jenkins CLI>> which provides a command
+to install plugins.
+
+
+Scripts to manage Jenkins environments, or configuration management code, may
+need to install plugins without direct user interaction in the web UI.
+
+The Jenkins CLI allows a command line user or automation tool to download a
+plugin and its dependencies.
+
+[source]
+----
+java -jar jenkins-cli.jar -s http://localhost:8080/ install-plugin SOURCE ... [-deploy] [-name VAL] [-restart]
+
+Installs a plugin either from a file, an URL, or from update center.
+
+ SOURCE    : If this points to a local file, that file will be installed. If
+             this is an URL, Jenkins downloads the URL and installs that as a
+             plugin.Otherwise the name is assumed to be the short name of the
+             plugin in the existing update center (like "findbugs"),and the
+             plugin will be installed from the update center.
+ -deploy   : Deploy plugins right away without postponing them until the reboot.
+ -name VAL : If specified, the plugin will be installed as this short name
+             (whereas normally the name is inferred from the source name
+             automatically).
+ -restart  : Restart Jenkins upon successful installation.
+----
+
+
+=== Advanced installation
+
+The Update Center only allows the installation of the most recently released
+version of a plugin. In cases where an older release of the plugin is desired,
+a Jenkins administrator can download an older `.hpi` archive and manually
+install that on the Jenkins master.
+
+==== From the web UI
+
+Assuming a `.hpi` file has been downloaded, a logged-in Jenkins administrator
```

Comment:
Depending on whether you read this as 'dot hpi' or just 'hpi,' you may want to s/a/an/